### PR TITLE
Follow/unfollow

### PR DESCRIPTION
**Pull request by dipen2000**
Please review my follow / unfollow feature of the social media app

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,19 @@
...
+import { useSelector } from "react-redux";
 
 function App() {
+  const users = useSelector((state) => state.user.users);
+  // const catwomanObj = users?.find((dbUser) => {
+  //   dbUser?.username === "the_cat";
+  // });
+  // console.log(users[users?.length - 2]);
+  // console.log(users[0]);
+  // console.log(catwomanObj);
```
Comment (SinchanaMS) [inline src/App.js:12]: Remove commented code

File: src/Pages/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import { Sidebar } from "../../Components/Sidebar/Sidebar";
+import { RightSidebar } from "../../Components/RightSidebar/RightSidebar";
+import { SortBar } from "../../Components/SortBar/SortBar";
+import { PostCard } from "../../features/post";
+import { NewPost } from "../../features/post";
+import { useEffect } from "react";
+import { getPosts } from "../../features/post";
+import { getAllUsers } from "../../features/user";
+import { useDispatch, useSelector } from "react-redux";
+import { getLoggedInUserFeed } from "../../Utils";
```
Comment (SinchanaMS) [inline src/Pages/Home/Home.js:12]: Use absolute paths and barrels for imports.

**Review by SinchanaMS — APPROVED** [review]
The code looks good, working as expected!

File: src/Pages/SinglePage/SinglePage.js [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+                      <div className="flex-row align-center-flex gap-1">
+                        <input
+                          className="flex-grow-1 reply-input-box"
+                          type="text"
+                          placeholder="Write your reply here."
+                          name="reply"
+                          onChange={commentChangeHandler}
+                          value={comment.reply}
+                        />
+                        <ButtonPrimary disabled={!comment?.reply.trim()}>
```
Comment (Shivani-1524) [inline src/Pages/SinglePage/SinglePage.js:222]: nice use of trim

**Review by Shivani-1524 — COMMENTED** [review]
looks good to merge after looking into the other reviews

**Pull request merged**